src/schemaComponents/templates.tsx

```diff
@@ -0,0 +1,18 @@
+import { JSONSchema7Definition } from 'json-schema';
+
+const newProperties = (title: string): JSONSchema7Definition => {
```

Comment:
Should be named `stringDefinition` nope ?

Reply:
Indeed, some leftovers of my tests